src/main/java/net/dv8tion/jda/core/managers/GuildController.java

```diff
@@ -377,19 +387,25 @@ protected void handleResponse(Response response, Request<Integer> request)
      *             <br>See {@link net.dv8tion.jda.core.utils.PermissionUtil#canInteract(Member, Member) PermissionUtil.canInteract(Member, Member)}</li>
      *         </ul>
      *
-     * @return {@link net.dv8tion.jda.core.requests.RestAction RestAction}
+     * @return {@link net.dv8tion.jda.core.requests.restaction.AuditableRestAction AuditableRestAction}
      *         Kicks the provided Member from the current Guild
      */
-    public RestAction<Void> kick(Member member)
+    public AuditableRestAction<Void> kick(Member member, String reason)
```

Comment:
Client accounts cannot use the `X-Audit-Log-Reason` header that is why I added kick/ban with the query parameter.